test/boilerplate_suggestors/props_meta_migrator_test.dart

```diff
@@ -20,25 +20,26 @@ import '../util.dart';
 
 main() {
   group('PropsMetaMigrator', () {
-    final testSuggestor = getSuggestorTester(PropsMetaMigrator());
+    final converter = ClassToMixinConverter();
+    final testSuggestor = getSuggestorTester(PropsMetaMigrator(converter));
 
     tearDown(() {
-      propsAndStateClassNamesConvertedToNewBoilerplate = {};
+      converter.setConvertedClassNames({});
```

Comment:
#nit Instead of sharing this converter and having to worry about cleaning up after it, it'd be nice to have a fresh instance each time.